### PR TITLE
feature/392305 - Cookie page content

### PR DESCRIPTION
**Pull request by davidjamesstone**
Update cookie page content

### CODE REVIEW

**Review by alexluckett — APPROVED** [review]

File: src/server/views/help/cookies.html [hunk 1]
```diff
@@ -2,37 +2,40 @@
...
-                    <td class="govuk-table__cell">crumb</td>
-                    <td class="govuk-table__cell">Helps us to prevent cross site scripting attacks</td>
-                    <td class="govuk-table__cell">When you close your browser</td>
-                </tr>
-            </tbody>
-        </table>
-    </div>
+    <h2 class="govuk-heading-m">Essential cookies</h2>
+    <p class="govuk-body">Essential cookies keep your information secure. We do not need to ask your permission to
+      use them.</p>
```
Comment (colinrotherham) [inline src/server/views/help/cookies.html:17]: Can you double check your hard wrapping settings?

Not necessarily for paragraph text, but HTML is sensitive to empty space

File: src/server/views/help/cookies.html [hunk 2]
```diff
@@ -2,37 +2,40 @@
...
 {% block content %}
 <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-l">Cookies on {{ serviceName }}</h1>
-        <p class="govuk-body">Cookies are files saved on your phone, tablet or computer when you visit a website.</p>
-        <p class="govuk-body">We use cookies to remember information you've entered when applying to prove your eligibility.</p>
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Cookies</h1>
+    <p class="govuk-body">Cookies are small files saved on your phone, tablet or computer when you visit a website.
+    </p>
```
Comment (colinrotherham) [inline src/server/views/help/cookies.html:12]:
```suggestion
    <p class="govuk-body">Cookies are small files saved on your phone, tablet or computer when you visit a website.</p>
```

File: src/server/views/help/cookies.html [hunk 3]
```diff
@@ -2,37 +2,40 @@
...
+      <thead>
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col">Name</th>
+          <th class="govuk-table__header" scope="col">Purpose</th>
+          <th class="govuk-table__header" scope="col">Expires</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell">session</td>
```
Comment (colinrotherham) [inline src/server/views/help/cookies.html:28]: This was missed from the previous HTML but can we fix an accessibility issue?

Without the [`govukTable()` macro](https://design-system.service.gov.uk/components/table/) it's easy to miss cookie names “session” and “crumb” as headings

Unlike the column headings they'll need `scope="row"`

```suggestion
          <th scope="row" class="govuk-table__header">session</th>
```

File: src/server/views/help/cookies.html [hunk 4]
```diff
@@ -2,37 +2,40 @@
...
+          <td class="govuk-table__cell">Remembers the information you enter</td>
+          <td class="govuk-table__cell">When you close the browser, or after 24 hours</td>
+        </tr>
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell">crumb</td>
```
Comment (colinrotherham) [inline src/server/views/help/cookies.html:33]:
```suggestion
          <th scope="row" class="govuk-table__header">crumb</th>
```

**Review by colinrotherham — APPROVED** [review]

**Pull request merged**